src/index.js

```diff
@@ -1,2 +1,39 @@
-// import files
+import './style.css';
+import {
+  createNewProject, editProjects, showProjectItems,
+} from './controller';
+import { myProjects } from './project-model';
 
+localStorage.setItem('projects', JSON.stringify(myProjects));
+
+document.addEventListener('DOMContentLoaded', () => {
+  // console.log('lstorage');
```

Comment:
- [ ] [REQUIRED] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))